src/main/java/com/hibernate/Main.java

```diff
@@ -42,7 +41,8 @@ public static void main(String[] args) {
 
         LocalDate requestedDate = LocalDate.now();
         System.out.println("Requested date: " + requestedDate);
-        List<MovieSession> res = movieSessionService.findAvailableSessions(1L, requestedDate);
-        System.out.println(res);
+        List<MovieSession> availableSessions = movieSessionService
+                .findAvailableSessions(movie.getId(), requestedDate);
+        System.out.println(availableSessions);
```

Comment:
what gets printed out here? can you please share the screenshot?

Reply:
![image](https://user-images.githubusercontent.com/75170409/106564141-7ffea900-6535-11eb-9dab-322342336a0d.png)